crypto/cms/cms_lib.c

```diff
@@ -74,11 +74,7 @@ CMS_ContentInfo *CMS_ContentInfo_new(void)
 
 void CMS_ContentInfo_free(CMS_ContentInfo *cms)
 {
-    if (cms != NULL) {
-        ossl_cms_env_enc_content_free(cms);
```

Comment:
If we are going to remove this call, then this ossl function should be removed (It was the only place it was called). 
Most of this CMS code is a horrible mess. If we are doing post_free's it might have made sense to NULL them after doing a free. That way if anothe call is done in the cleanup it can be there without being an issue.

Reply:
removed...